pkg/daemon/update.go

```diff
@@ -714,8 +715,18 @@ func (dn *Daemon) update(oldConfig, newConfig *mcfgv1.MachineConfig, skipCertifi
 		return err
 	}
 
+	crioOverrideConfigmapExists := false
```

Comment:
Could we potentially make this into its own function? Also, I think this logic might flow a little better:

```golang
func (dn *Daemon) hasImageRegistryDrainOverrideConfigMap() (bool, error) {
	_, err := dn.kubeClient.CoreV1().ConfigMaps(ctrlcommon.MCONamespace).Get(context.TODO(), constants.ImageRegistryDrainOverrideConfigmap, metav1.GetOptions{})
	if err == nil {
		return true, nil
	}

	if apierrors.IsNotFound(err) {
		return false, nil
	}

	return false, fmt.Errorf("Error fetching image registry drain override configmap: %w", err)
}